client/signup/steps/user/index.jsx

```diff
@@ -594,6 +617,26 @@ export class UserStep extends Component {
 		);
 	}
 
+	getCustomizedActionButtons() {
+		if ( this.props.isSocialFirst ) {
+			return (
+				<Button
+					className="step-wrapper__navigation-link forward"
+					href={ this.getLoginUrl() }
+					variant="link"
+				>
+					<span>{ this.props.translate( 'Log in' ) }</span>
+				</Button>
+			);
+		}
+	}
+
+	getIsSticky() {
+		if ( this.props.isSocialFirst ) {
+			return false;
+		}
+	}
```

Comment:
This function always returns `falsy`. Is that intentional?

Reply:
Yes! Without this, [this line](https://github.com/Automattic/wp-calypso/blob/trunk/client/signup/step-wrapper/index.jsx#L206) would set the `sticky` prop on `ActionButtons` to null,  positioning the Log In custom button to the bottom on mobile:

![image](https://github.com/Automattic/wp-calypso/assets/52076348/7c8266fc-7dcd-4d21-8579-9d7a1b6dc8c5)

Reply:
Can we make it explicit and return `false` in other cases?